src/Core/AdminConsole/Services/Implementations/RabbitMqEventHttpPostListener.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Text;
+using System.Text.Json;
+using Bit.Core.Models.Data;
+using Bit.Core.Settings;
+using Microsoft.Extensions.Logging;
+
+namespace Bit.Core.Services;
+
+public class RabbitMqEventHttpPostListener : RabbitMqEventListenerBase
+{
+    private readonly string _httpPostUrl;
+
+    protected override string QueueName => "events-httpPost-queue";
+
+    public RabbitMqEventHttpPostListener(
+        ILogger<RabbitMqEventListenerBase> logger,
+        GlobalSettings globalSettings)
+        : base(logger, globalSettings)
+    {
+        _httpPostUrl = globalSettings.RabbitMqHttpPostUrl;
+    }
+    protected override async Task HandleMessageAsync(EventMessage eventMessage)
+    {
+        var message = JsonSerializer.Serialize(eventMessage);
+        using var httpClient = new HttpClient();
+        var content = new StringContent(message, Encoding.UTF8, "application/json");
+        var response = await httpClient.PostAsync(_httpPostUrl, content);
```

Comment:
There is a special `HttpContent` class for JSON already.

```suggestion
        using var httpClient = new HttpClient();
        var content = JsonContent.Create(eventMessage);
        var response = await httpClient.PostAsync(_httpPostUrl, content);
```